docs/production/multiple-organizations.md

```diff
@@ -45,6 +45,20 @@ things:
   authentication method, review
   [the notes on `SOCIAL_AUTH_SUBDOMAIN` below](#authentication).
 
+## Enabling open realm creation
+
+If you want anyone to be able to create new realms on your server, you
+can enable open realm creation. This will add a **Create new
+organization** link to your Zulip homepage footer, and anyone will be able to
+create a new realm by visiting the `/new/` endpoint. This
+feature is disabled by default in production instances, and can be
+enabled by setting `OPEN_REALM_CREATION = True` in `/etc/zulip/settings.py`.
+
+This setting should be used with caution if your server is available on the
+public internet, since it opens up the ability for anyone to create subdomains
+on your server, with the resulting organizations being under their ownership -
+which is likely an undesirable scenario in the case of unauthorized, unknown parties.
+
```

Comment:
This should probably be split up into a couple of sentences to be easier to parse; @timabbott maybe you can tweak it a bit when merging?